windows/src/Makefile

```diff
@@ -234,16 +244,6 @@ web-install:
 
 # ----------------------------------------------------------------------
 
-disable-search:
-    -net stop wsearch
-    -net stop MsMpSvc
-
-enable-search:
-    -net start wsearch
-    -net start MsMpSvc
```

Comment:
I assume these are no longer used, then?  Not that I understand what they were doing at first, but I'm guessing this was in place as a preliminary measure to facilitate branding that's no longer sufficient/accurate?

Reply:
These are not used. They were there from a long time ago when Windows Search and the Microsoft Anti-Malware Service (name?) would lock files when we were working with them and cause the build to randomly fail. Nowadays, Windows Search is better behaved and we exclude the anti-malware services from scanning Keyman project folders to avoid the issue.

Reply:
Wait... so we block anti-malware services from checking Keyman resources?  I assume that's just during installation?

Reply:
No that's just in the build process. anti-malware checks the .exes we create and locks them, then we try to codesign while the file is locked and everything falls over. Or similar.

But now we just configure our dev machines to exclude the Keyman source/build folders from scanning and that avoids the issue. Hence why I deleted these lines from the Makefile :)